src/multiparser.js

```diff
@@ -67,9 +67,10 @@ function fromBabylonFlowOrTypeScript(path) {
        * gql`...`
        */
       if (
+        // We currently don't support expression inside GraphQL template literals
+        parent.quasis.length === 1 &&
```

Comment:
You don't necessarily need to make this change, but it might be more readable if we checked the expressions length instead of the quasis length (either works, though, since they're associated with each other).

Reply:
Indeed